README.md

```diff
@@ -8,7 +8,8 @@ Instead of each user specifying what they want to get notified about; or having
 managed by an admin, this plugin allows setting notification rules in the pipeline configuration via environment variables.
 
 The following plugins exist:
-- gong-notifier-email: Sends E-Mail notifications
+- [gong-notifier-email](gong-notifier-email): Sends E-Mail notifications
```

Comment:
Did you check MarkDownlint?
https://github.com/DavidAnson/markdownlint

The reason I ask is I usually find that helpful, and the current READMEs actually look like compatible with those suggestions. :)

Reply:
Thanks for the tip! I installed the VSCode extension and fixed all the issues in the READMEs.

Reply:
Yeah I also use it form VSCode :)